Automattic/wp-calypso
# Earn Section: Update landing page

This is is the main landing page for the Earn section using the new card layout. It's still WIP and this is a skeleton version with a couple of cards, but it will form the basis of the section as a whole.

#### Testing instructions

Use Calypso live, and go to the `/earn/` you should be presented with the new card layout. Only the recurring payments button currently works.


Fixes #34517 

Some of the bits left to do are:

- [x] Update SVGs
- [x] Include the other cards and copy
- [x] Increase the default size of the main div (it looks a bit narrow)
- [x] Add the click handlers for the CTA buttons on each card, both for upgrading and to go to the appropriate section
- [ ] Add the analytics
- [x] Find the most appropriate way of not having a Learn More link on Simple Payments
- [ ] Work out the best way to detect the activation of each feature
- [ ] Provide the 3rd variation of each card when the feature is activated
